sqblt.code-workspace

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This file is only relevant to your local repository (on your computer's hard drive/SSD). It should be removed from the pull request.

```she
git rm --cached sqblt.code-workspace
git commit
```

This will instruct Git to stop tracking the file, but leave it on your computer.

Reply:
ok, did this

Reply:
Probably need to run `git push` still; not seeing this change on the pull request.